### PR TITLE
10/19 updates (redo)

### PR DESCRIPTION
**Pull request by emilysaeli**
Opening this because I accidentally merged #29 before it was ready.

### CODE REVIEW

File: Whiteboard design session/WDS trainer guide - Azure Stack.md [hunk 1]
```diff
@@ -232,6 +246,8 @@ Contoso is looking for FT to provide the following for their expansion into Nort
 
 4.  One of the key reasons Contoso wants to go to the cloud is to take advantage of tools and services for automated deployments and application development. Will Azure Stack make it to where we must use two skillsets?
 
+5.  Fabrikam has already its own Azure Active Directory tenant. Will it be necessary to create duplicate accounts for Fabrikam users?
```
Comment (rdendtler) [inline Whiteboard design session/WDS trainer guide - Azure Stack.md]: Correct order of words.. "Fabrikam already has ...."

File: Hands-on lab/HOL step-by-step - Azure Stack.md [hunk 2]
```diff
@@ -139,18 +139,18 @@ Duration: 15-30 minutes
 
     ![Screenshot of the Create Quota blade.](images/Hands-onlabstep-by-step-AzureStackimages/media/image77.png)
 
-8. Click **Review + Create** and then confirm the creation. 
+8. select **Review + Create** and then confirm the creation. 
 
     ![OK is selected in the Quotas blade.](images/Hands-onlabstep-by-step-AzureStackimages/media/image79.png)
 
-9. The Plan will deploy immediately. Click **+ Create a resource** in the Azure Stack Admin portal.
+9. The Plan will deploy immediately. select **+ Create a resource** in the Azure Stack Admin portal.
```
[the remark this answers is not in the document]
Comment (paulopsgility) [reply Hands-on lab/HOL step-by-step - Azure Stack.md]: Resolved

File: Hands-on lab/HOL step-by-step - Azure Stack.md [hunk 3]
```diff
@@ -115,21 +115,21 @@ Duration: 15-30 minutes
 
         ![New plan blade, fields are set to the previously defined settings.](images/Hands-onlabstep-by-step-AzureStackimages/media/image71.png)
 
-4.  Click **Services**.
+4.  select **Services**.
```
[the remark this answers is not in the document]
Comment (paulopsgility) [reply Hands-on lab/HOL step-by-step - Azure Stack.md]: Resolved

File: Hands-on lab/Before the HOL - Azure Stack - Operations.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+The names of manufacturers, products, or URLs are provided for informational purposes only and Microsoft makes no representations and warranties, either expressed, implied, or statutory, regarding these manufacturers or the use of the products with any Microsoft technologies. The inclusion of a manufacturer or product does not imply endorsement of Microsoft of the manufacturer or product. Links may be provided to third party sites. Such sites are not under the control of Microsoft and Microsoft is not responsible for the contents of any linked site or any link contained in a linked site, or any changes or updates to such sites. Microsoft is not responsible for webcasting or any other form of transmission received from any linked site. Microsoft is providing these links to you only as a convenience, and the inclusion of any link does not imply endorsement of Microsoft of the site or the products contained therein.
+
+© 2018 Microsoft Corporation. All rights reserved.
```
[the remark this answers is not in the document]
Comment (paulopsgility) [reply Hands-on lab/Before the HOL - Azure Stack - Operations.md]: resolved

File: Whiteboard design session/WDS trainer guide - Azure Stack.md (hunk 1 above)
Comment (paulopsgility) [reply Whiteboard design session/WDS trainer guide - Azure Stack.md]: resolved

File: Hands-on lab/HOL step by step - Azure Stack - Operations.md [hunk 5]
```diff
@@ -0,0 +1,1488 @@
...
+
+     ```
+     Invoke-WebRequest `
+  		-Uri https://raw.githubusercontent.com/polichtm/MCW-Azure-Stack/master/Hands-on%20lab/resources.operations/ContosoWebAppTempate.json `
+		-OutFile 'C:\Downloads\Azure Stack Marketplace Item Generator and Sample\ContosoWebAppTemplate\DeploymentTemplates\ContosoWebAppTempate.json'
+     ```
+
+1.  In File Explorer, navigate to the C:\Downloads\Azure Stack Marketplace Item Generator and Sample\ContosoWebAppTemplate\strings folder.
+
+1.  Open the file **resources.resjson** in Notepad, modify its content so it matches the following, save, and close the file:
```
[the remark this answers is not in the document]
Comment (paulopsgility) [reply Hands-on lab/HOL step by step - Azure Stack - Operations.md]: resolved

File: Hands-on lab/HOL step by step - Azure Stack - Operations.md [hunk 6]
```diff
@@ -0,0 +1,1488 @@
...
+
+Contoso has expressed to FT the need to embrace Microsoft Azure technologies as well as technologies that will help their organization with a more agile continuous integration and continuous deployment model for application deployment. Contoso also underscored the need for cooperation with Fabrikam's integration teams, including the intent to delegate some of the infrastructure management tasks. 
+
+With these goals in mind, Contoso has challenged FT to help implement the hosted environment must also accommodate requirements regarding integration work to be carried out by Fabrikam. In addition, as internal workloads are transitioned to the hosting environment, Contoso internal audit team must retain its ability to track all of the infrastructure changes. For compliance purposes, the delegation model that will provide Contoso and Fabrikam staff with insight into the hosted environment must comply with the principle of least privilege. To satisfy Contoso governance requirements, FT must document standard operating procedures that will be carried out within the hosted infrastructure.
```
[the remark this answers is not in the document]
Comment (paulopsgility) [reply Hands-on lab/HOL step by step - Azure Stack - Operations.md]: resolved

**Pull request merged**